README.md

```diff
@@ -22,8 +22,8 @@ Functions
 `nba_scraper`
 
 -   Scrapes data from ESPN NBA data into a csv file. User can specify
-    the year of the season (2016, 2017, etc) and the season type
-    (regular or playoffs).
+    the year of the season (2001 to 2019 at the moment) and the season
+    type (“regular” or “playoffs”).
```

Comment:
```suggestion
    type (“regular” or “postseason”).
```

Reply:
this has been changed accordingly with a more recent commit!